packages/react/src/analytics/integrations/Forter/__tests__/Forter.test.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  EventTypes,
+  SetUserEventData,
+  TrackTypes,
+  TrackTypesValues,
+  utils,
+} from '@farfetch/blackout-analytics';
+import { ForterTokenTid } from '../constants';
+import {
+  loadIntegrationData,
+  trackEventsData,
+} from 'tests/__fixtures__/analytics';
+import { postTracking } from '@farfetch/blackout-client';
+import Forter from '..';
+const eventData = trackEventsData[EventTypes.CHECKOUT_STARTED];
+
+let mockPostTrackingsPromiseResolve: (value?: SetUserEventData) => void;
+const mockPostTrackingsPromise = new Promise(
+  resolve => (mockPostTrackingsPromiseResolve = resolve),
+);
+const mockSiteId = 'c3f674e6511e';
+const mockOrigin = 'Whitelabel Portal';
+const mockForterToken =
+  '44807307964b4931a89bf364e580bdc5_1594724271711__UDF43_9ck';
+const mockForterTokenLoadedPromise = Promise.resolve(mockForterToken);
+
+// @ts-expect-error
```

Comment:
Why is this `@ts-expect-error` necessary? It is a good practice to describe why you needed to have it so we can understand why you needed to do so. If I remove this the only error I got was a missing import of the `StrippedDownAnalytics` type that when adding an import to it, cleared the error.